packages/components/src/higher-order/with-focus-outside/test/index.tsx

```diff
@@ -92,32 +92,4 @@ describe( 'withFocusOutside', () => {
 
 		await waitFor( () => expect( onFocusOutside ).toHaveBeenCalled() );
 	} );
-
-	it( 'should not call handler if focus shifts outside the component when the document does not have focus', async () => {
-		render( <TestComponent onFocusOutside={ onFocusOutside } /> );
-
-		// Force document.hasFocus() to return false to simulate the window/document losing focus
-		// See https://developer.mozilla.org/en-US/docs/Web/API/Document/hasFocus.
-		document.hasFocus = () => false;
-
-		const input = screen.getByRole( 'textbox' );
-		input.focus();
-		input.blur();
-
-		await waitFor( () => expect( onFocusOutside ).not.toHaveBeenCalled() );
-	} );
-
-	it( 'should cancel check when unmounting while queued', async () => {
```

Comment:
This is the behavior we're changing. We want the callback because focus is moved outside the component.